test/cets_SUITE.erl

```diff
@@ -0,0 +1,259 @@
+-module(cets_SUITE).
+-include_lib("common_test/include/ct.hrl").
+ 
+-compile([export_all, nowarn_export_all]).
+ 
+all() -> [test_multinode, node_list_is_correct,
+          test_multinode_auto_discovery, test_locally,
+          handle_down_is_called,
+          events_are_applied_in_the_correct_order_after_unpause,
+          pause_multiple_times,
+          unpause_twice,
+          write_returns_if_remote_server_crashes,
+          mon_cleaner_works, sync_using_name_works,
+          insert_many_request].
+ 
+init_per_suite(Config) ->
+    Node2 = start_node(ct2),
+    Node3 = start_node(ct3),
+    Node4 = start_node(ct4),
+    [{nodes, [Node2, Node3, Node4]}|Config].
+
+end_per_suite(Config) ->
+    Config.
+
+init_per_testcase(test_multinode_auto_discovery, Config) ->
+    ct:make_priv_dir(),
+    Config;
+init_per_testcase(_, Config) ->
+    Config.
+ 
+end_per_testcase(_, _Config) ->
+    ok.
+ 
+test_multinode(Config) ->
```

Comment:
This test checks a few thing, e.g. joining nodes (both before and after adding data), inserting values, deleting values, bulk operations. Could you split it to minimal test cases where each one tests exactly one thing?

Reply:
I added more tests for each operation